src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Int8Array.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace System.Runtime.InteropServices.JavaScript
+{
+    [CLSCompliant(false)]
+    public sealed class Int8Array : TypedArray<Int8Array, sbyte>
+    {
+        public Int8Array()
+        { }
+
+        public Int8Array(int length) : base(length)
+        { }
+
```

Comment:
Nit:
Another extra blank line.  I'll stop commenting on these, but there look to be a few more.
```suggestion
```